tests/baselines/reference/privateNameAndAny.js

```diff
@@ -1,8 +1,12 @@
 //// [privateNameAndAny.ts]
 class A {
-    #foo = true; 
+    #foo = true;
+    static #baz = 10;
+    static #m() {}
     method(thing: any) {
         thing.#foo; // OK
+        thing.#m();
+        thing.#bar;
```

Comment:
Is there a reason to duplicate `thing.#bar`?

Reply:
It was supposed to be `#baz`. Thanks.

Reply:
it was supposed to be `#baz`. Fixed